analytics/cdk/emr-eks/bin/emr-eks.ts

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import EmrEksStack from '../lib/emr-eks-stack';
```

Comment:
This import does not work since the file is called `../lib/emr-eks-blueprint-stack.ts`.